packages/hebao_v1/contracts/base/ControllerImpl.sol

```diff
@@ -65,19 +69,27 @@ contract ControllerImpl is Claimable, Controller
         quotaManager = _quotaManager;           // modifiable
     }
 
+    function setCollectTo(address _collectTo)
+        external
+        onlyOwner
+    {
+        collectTo = _collectTo;
```

Comment:
`collectTo` could be set to `address(0)`. Not necessarily wrong, but `init` does have the check. Either is fine for me.

Reply:
fixed